include/bluetooth/conn.h

```diff
@@ -367,6 +367,19 @@ struct bt_conn_remote_info {
 	};
 };
 
+/** LE Transmit Power Level Structure */
+struct bt_conn_le_tx_power {
+
+	/* Input: 1M, 2M, Coded S2 or Coded S8 */
+	uint8_t phy;
+
+	/* Output: current transmit power level */
+	int8_t current_level;
+
+	/* Output: maximum transmit power level */
+	int8_t max_level;
```

Comment:
Shouldn't the individual struct members also have doxygen formatted documentation?